pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -370,10 +370,12 @@ message LinuxContainerConfig {
     optional LinuxContainerResources resources = 1;
     // Capabilities to add or drop.
     optional Capability capabilities = 2;
-    // Optional SELinux context to be applied.
-    optional SELinuxOption selinux_options = 3;
+    // Optional SELinux label for container processes
+    optional SELinuxOption selinux_process_label = 3;
+    // Optional SELinux mount label for container mount points
+    optional SELinuxOption selinux_mount_label = 4;
     // User contains the user for the container process.
-    optional LinuxUser user = 4;
+    optional LinuxUser user = 5;
```

Comment:
@yujuhong That was my reasoning as well. Might as well make changes now rather than later.